cmd/world/cardinal/dev.go

```diff
@@ -182,165 +116,182 @@ var devCmd = &cobra.Command{
 /////////////////
 
 func init() {
-	devCmd.Flags().Bool(flagWatch, false, "Dev mode with hot reload support")
-	devCmd.Flags().Bool(flagNoEditor, false, "Run Cardinal without the editor")
+	devCmd.Flags().Bool(flagNoEditor, false, "Run Cardinal without the editor (default: false)")
+	devCmd.Flags().Bool(flagPrettyLog, true, "Run Cardinal with pretty logging (default: true)")
```

Comment:
```suggestion
	devCmd.Flags().Bool(flagNoEditor, false, "Run Cardinal without the editor (default false)")
	devCmd.Flags().Bool(flagPrettyLog, true, "Run Cardinal with pretty logging")
```

default true printed twice on `--pretty-log` flag

![Screenshot from 2024-05-18 01-30-56.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/YO1Dcg4NByYdZHvKXaTq/020f8e45-e99d-4b30-8c8c-d266ee712c44.png)